src/editor/QueryField.tsx

```diff
@@ -222,8 +249,8 @@ class QueryField extends React.Component<any, any> {
     return change || this.state.value.change();
   };
 
-  applyTypeahead = (editor: CoreEditor, suggestion: { text: any; type: string; deleteBackwards: any }): { value: object } => {
-    return { value: new Value() };
+  applyTypeahead = (editor: CoreEditor, suggestion: Suggestion): CoreEditor => {
+    return this.editorEl();
```

Comment:
This function no longer does anything.

Reply:
Nope, and I don't think it did anything before, and is just there to be overrideable so it didn't really matter what was there, accept that I wanted the function declaration and return value to be the same, and so decided to return this.editorEl()